lock/src/main/java/com/auth0/android/lock/provider/OAuth2WebAuthProvider.java

```diff
@@ -137,31 +139,44 @@ private void startAuthorization(Activity activity, Uri authorizeUri, String conn
     @Override
     public boolean authorize(Activity activity, @NonNull AuthorizeResult data) {
         if (!data.isValid(OAUTH2_REQUEST_CODE)) {
+            Log.w(TAG, "Invalid request.");
```

Comment:
Probably yes if it has more meaningful info, let's avoid adding logs without any useful info, it's just noise